backend/src/controllers/banner.controller.js

```
@@ -54,11 +97,12 @@ class BannerController {
       if (!deletionResult) {
-        return res
-          .status(400)
-          .json({
-            errors: [{ msg: "Banner with the specified id does not exist" }],
-          });
+        return res.status(400).json({
+          errors: [{ msg: "Banner with the specified id does not exist" }],
+        });
+        return res.status(400).json({
+          errors: [{ msg: "Banner with the specified id does not exist" }],
+        });
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)